packages/mdc-form-field/README.md

```diff
@@ -7,141 +7,95 @@ path: /catalog/input-controls/form-fields/
 
-```javascript
-const MDCFormField = mdc.formField.MDCFormField;
-const MDCFormFieldFoundation = mdc.formField.MDCFormFieldFoundation;
-```
+### JavaScript Instantiation
 
-### Instantiation
+If you are using MDC Form Field with an MDC Web component that has a [ripple](../mdc-ripple) effect, you can instantiate `MDCFormField` and set its [`input` property](#MDCFormField-properties-and-methods) to activate the ripple effect upon interacting with the label. Here is an example using [MDC Radio](../mdc-radio):
 
-```javascript
+```js
 import {MDCFormField} from '@material/form-field';
+import {MDCFormField} from '@material/radio';
 
 const formField = new MDCFormField(document.querySelector('.mdc-form-field'));
+const radio = new MDCRadio(document.querySelector('.mdc-radio'));
+formField.input = radio;
 ```
 
-### MDCFormField API
+> See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
 
-The `MDCFormField` functionality is exposed through a single accessor method.
+## Variants
 
-#### MDCFormField.input
+### Label position
 
-Read-write property that works with an instance of an MDC Web input element.
+By default, the input will be positioned before the label. You can position the input after the label by adding the `mdc-form-field--align-end` class:
 
-In order for the label ripple integration to work correctly, this property needs to be set to a
-valid instance of an MDC Web input element which exposes a `ripple` getter.
+```html
+<div class="mdc-form-field mdc-form-field--align-end">
+  <input type="checkbox" id="input">
+  <label for="input">Label</label>
+</div>
+```
 
-```javascript
-const formField = new MDCFormField(document.querySelector('.mdc-form-field'));
-const radio = new MDCRadio(document.querySelector('.mdc-radio'));
+## `MDCFormField` Properties and Methods
 
-formField.input = radio;
-```
+Property | Value Type | Description
+--- | --- | ---
+`input` | String | Gets and sets the form field input. 
+
+In order for the label ripple integration to work correctly, this property needs to be set to a valid instance of an MDC Web input element which exposes a `ripple` getter. No action is taken if the `input` property is not set or the input instance doesn't expose a `ripple` getter.
 
-No action is taken if the `input` property is not set or the input instance doesn't expose a
-`ripple` getter.
+## Usage within Web Frameworks
 
+If you are using a JavaScript framework, such as React or Angular, you can create a Form Field for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../docs/integrating-into-frameworks.md).
 
-### Adapter
+### `MDCFormFieldAdapter`
```

Comment:
I'm not sure why the table below is empty, but form field's adapter actually has 4 APIs. Can you fill this in?

Reply:
Oh there's stuff at the bottom, it's just collapsed on the GitHub PR diff view because nothing changed.

Reply:
Oh, whoops, dunno how I didn't notice the fold.